gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -372,6 +392,34 @@ MM_VerboseHandlerOutputStandard::handleCompactEndInternal(MM_EnvironmentBase* en
 
 #endif /* defined(OMR_GC_MODRON_COMPACTION) */
 
+void
+MM_VerboseHandlerOutputStandard::handleGlobalMarkEndNoLock(J9HookInterface** hook, uintptr_t eventNum, void* eventData)
+{
+	MM_ConcurrentPhaseEndEvent *event = (MM_ConcurrentPhaseEndEvent *)eventData;
+	MM_ConcurrentPhaseStatsGlobalMarkStandard *stats = (MM_ConcurrentPhaseStatsGlobalMarkStandard *)event->concurrentStats;
+	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
+	MM_VerboseManager* manager = getManager();
+	MM_VerboseWriterChain* writer = manager->getWriterChain();
+
+	uint64_t currentTime = event->timestamp;
+	uint64_t previousTime = manager->getLastConcurrentGCTime();
+	manager->setLastConcurrentGCTime(currentTime);
```

Comment:
`setLastConcurrentGCTime` is already done in `handleConcurrentCollectionStart` so it shouldn't be done again. You can let `handleConcurrentCollectionStart` do it and remove it from here, make a comment here after removing it.

Reply:
Also we're just using the previous time to detect clock error, we don't report an interval time like  `handleConcurrentCollectionStart`. So best not to record the previous time here